warn/scrapers/ga.py

```diff
@@ -0,0 +1,88 @@
+import re
+import typing
+from datetime import datetime
+from pathlib import Path
+
+from bs4 import BeautifulSoup
+
+from .. import utils
+from ..cache import Cache
+
+
+def scrape(
+    data_dir: Path = utils.WARN_DATA_DIR,
+    cache_dir: typing.Optional[Path] = utils.WARN_CACHE_DIR,
+) -> Path:
+    """
+    Scrape data from Georgia.
+
+    Keyword arguments:
+    data_dir -- the Path were the result will be saved (default WARN_DATA_DIR)
+    cache_dir -- the Path where results can be cached (default WARN_CACHE_DIR)
+
+    Returns: the Path where the file is written
+    """
+    cache = Cache(cache_dir)
+
+    area = 9  # statewide
+
+    current_year = datetime.now().year
+    first_year = 2002  # first available year
+
+    years = list(range(first_year, current_year + 1))
+    years.reverse()
+
+    # include column headers in first row
+    column_tags = ["td", "th"]
+
+    output_rows = []
+
+    for year in years:
+        url = f"https://www.dol.state.ga.us/public/es/warn/searchwarns/list?geoArea={area}&year={year}&step=search"
+
+        cache_key = f"ga/{year}.html"
+
+        if cache.exists(cache_key) and year < current_year:
+            html = cache.read(cache_key)
+        else:
+            # Get URL
+            page = utils.get_url(url)
+            html = page.text
+            cache.write(cache_key, html)
+
+        # Parse out data table
+        soup = BeautifulSoup(html, "html.parser")
+        table = soup.find_all(id="emplrList")  # output is list-type
+
+        # Loop through the table and grab the data
+        for table_row in table[0].find_all("tr"):
```

Comment:
```suggestion
        # Parse out data table
        soup = BeautifulSoup(html, "html.parser")
        table_list = soup.find_all(id="emplrList")  # output is list-type
        
        # We expect the first table to be there with our data
        assert len(table_list) > 0
        table = table_list[0]

        # Loop through the table and grab the data
        for table_row in table.find_all("tr"):
```
I'm riffing here, but what do you think about the idea of being a little more explicit about what we're expecting to parse, and then doing a little light test?